### ISSUE TITLE
uri-reference

### ISSUE THREAD

**Issue by drsm79** [koxudaxi/datamodel-code-generator#193]
**Describe the bug**

When parsing a jsonschema with a `uri-reference` you get a KeyError:

```
File "/mnt/c/Users/simon/code/py-oscal/venv/lib/python3.7/site-packages/datamodel_code_generator/parser/jsonschema.py", line 180, in <listcomp>
    for t in types
KeyError: 'uri-reference'
```

**To Reproduce**

Example schema:

https://github.com/usnistgov/OSCAL/blob/master/json/schema/oscal_component_schema.json

Used commandline:
```
$  datamodel-codegen --input-file-type jsonschema --input OSCAL/json/schema/oscal_component_schema.json
```

**Expected behavior**
Not throw an exception & create the objects.

**Version:**
 - OS: Windows 10
 - Python Verison: 3.7.8
 - datamodel-code-generator Version 0.5.24

**Additional context**

I've patched this in https://github.com/drsm79/datamodel-code-generator/commit/2821d2bd8bf959b9d3deb8ab88fe85a1fa3c8ab4 though I'm not sure if it should be a different type to URI. Happy to PR it.

**Comment by koxudaxi** [comment]
@drsm79 
Thank you for creating this issue.
It's a good suggestion :smile:  :+1: 

PR welcome to support `uri-reference`.

**Comment by drsm79** [comment]
Is there a path-like type it could be, else a string makes sense.

**Comment by drsm79** [comment]
Closed by #194
